cmd/kubectl-gadget/utils/trace.go

```diff
@@ -325,6 +664,13 @@ func genericStreamsDisplay(
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
 	completion := make(chan string)
 
+	client, err := k8sutil.NewClientsetFromConfigFlags(KubernetesConfigFlags)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error setting up Kubernetes client: %w", err)
```

Comment:
btw, the CI is failing in this line because "Fprintf call has error-wrapping directive %w".